cypress/e2e/WebInterface/Measure/Measure Group/MeasureGroupTabs.cy.ts

```diff
@@ -407,7 +408,7 @@ describe('Validating Stratification tabs', () => {
         cy.get(MeasureGroupPage.stratAssociationOne).should('have.attr', 'placeholder', 'Select All That Apply')
         //Association -- contains these values based off score type -- score type is Continuous Variable
         cy.get(MeasureGroupPage.stratAssociationOne).click()
-        cy.get('[class="base-Popper-root MuiAutocomplete-popper MuiAutocomplete-popperDisablePortal css-1pxin5q"]').each(($ele) => {
+        cy.get('[id="association-select-1-listbox"]').each(($ele) => {
```

Comment:
this is exactly what I was talking about. Thanks Brandon!!

Reply:
No problem
Apologies for not doing this the first go around.